src/Parsers/NullsAction.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+namespace DB
+{
+
+/// Some window functions support adding [IGNORE|RESPECT] NULLS
```

Comment:
Consider also adding keywords to client suggestions list
https://github.com/ClickHouse/ClickHouse/blob/001b67863a57c4d734d94e375e6fdeb4a9407951/src/Client/Suggest.cpp#L34-L36

Reply:
Also we may add them into fuzzer dictionary, but not sure if it's worth doing, since only one function supports it and we have queries in stateless tests for it that will be fuzzed.

Reply:
Truth be told, all aggregations support `IGNORE NULLS` (does nothing since it's the default), but only any/anyLast and its aliases support `RESPECT NULLS`. I'll look into adding them to the suggestion list.

Reply:
Seems simple enough

![Peek 29-11-2023 12-10](https://github.com/ClickHouse/ClickHouse/assets/664253/406f4cd1-9cb9-4a20-bc30-8396c2ba35c7)

Reply:
While introducing the fuzzer I noticed I'm not throwing if you pass `RESPECT NULLS` to normal (not aggregate) functions. Will fix it.